spring-cloud-kubernetes-fabric8-autoconfig/pom.xml

```diff
@@ -131,6 +131,15 @@
 			<scope>test</scope>
 		</dependency>
 
+		<dependency>
```

Comment:
fabric-uatoconfig tests where failing for me in a diff PR, and I have noticed that this dependency is needed, but it's not in place anymore. I guess it was coming from some spring-boot dependency and now it is not there anymore.

Reply:
All of our builds internally are passing both in the 2.1.x and main branches, so I don't think anything changes that would cause us to need this dependency now

Reply:
Im really sorry - the error was on my laptop, only. :(

Reply:
No worries glad you got if figured it.  Still worth making the other changes though.

Reply:
right, build is green now